README.md

```diff
@@ -2,69 +2,55 @@
 
 ## Overview
 
-BTP Manager is an operator for [SAP BTP Service Operator](https://github.com/SAP/sap-btp-service-operator) based on the [Kubebuilder](https://github.com/kubernetes-sigs/kubebuilder) framework. It extends Kubernetes API by providing [BtpOperator](https://github.com/kyma-project/btp-manager/blob/main/config/crd/bases/operator.kyma-project.io_btpoperators.yaml) CRD which allows you to manage SAP BTP Service Operator resource through CR. For more information, see the [BTP Manager documentation](./docs/README.md).
+BTP Manager is an operator for [SAP BTP Service Operator](https://github.com/SAP/sap-btp-service-operator) based on the [Kubebuilder](https://github.com/kubernetes-sigs/kubebuilder) framework. It extends Kubernetes API by providing [BtpOperator](https://github.com/kyma-project/btp-manager/blob/main/config/crd/bases/operator.kyma-project.io_btpoperators.yaml) CRD which allows you to manage SAP BTP Service Operator resource through CR. 
 
-## Prerequisites
+## Installation
+
+To enable the BTP Operator module from the latest release, you must install BTP Manager and SAP BTP Service Operator.
+
+### Prerequisites
 
 - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
-- Kubernetes cluster (you can use [k3d](https://k3d.io)) 
+- Kubernetes cluster, or [k3d](https://k3d.io) for local installation
 
-## Install BTP Manager
+> **CAUTION** You also need the `kyma-system` Namespace. If you don't have it in your cluster, use the following command to create it:
+> ```bash
+> kubectl create namespace kyma-system
+> ```
+
+### Steps
+ 
+1. To install BTP Manager, use the following commands:
+
+    ```bash
+    kubectl apply -f https://github.com/kyma-project/btp-manager/releases/latest/download/rendered.yaml
+    ```
+    > **TIP:** Use the same command to upgrade the module to the latest version.
+
+<br>
 
- You can install BTP Manager locally or in your cluster. For more information, read the [Install and uninstall BTP Manager](./docs/installation.md) document.
+ 2. To install SAP BTP Service Operator, apply the sample BtpOperator CR:
+
+    ```bash
+    kubectl apply -f https://raw.githubusercontent.com/kyma-project/btp-manager/main/config/samples/operator_v1alpha1_btpoperator.yaml
```

Comment:
It will install the resource in the default namespace. The intention was to have it in kyma-system (probably).